nengo/utils/builder.py

```diff
@@ -177,8 +177,8 @@ def _create_replacement_connection(c_in, c_out):
     return c
 
 
-def remove_passthrough_nodes(objs, connections,  # noqa: C901
-        create_connection_fn=_create_replacement_connection):
```

Comment:
And so changing the default to `None` and doing it in the function was just a side fix, and unrelated to flake8? That's fine, I'm just checking.